bigframes/pandas/__init__.py

```diff
@@ -652,7 +656,7 @@ def read_gbq_function(function_name: str):
 # Use __all__ to let type checkers know what is part of the public API.
 __all___ = [
     # Functions
-    "concat",
+    "clear_cache" "concat",
```

Comment:
Looks like we had some leftover code here to revert.

```suggestion
    "concat",
```

Reply:
Done.